src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,83 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+
+import duke.DukeException;
+import duke.parser.Parser;
+import duke.storage.TaskRecords;
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.Task;
+import duke.task.ToDos;
+import duke.ui.BotUI;
+
+/**
+ * Represents an adding command of task. A <code>AddCommand</code> object stores
+ * the details of the task BEFORE filtering/extracting the date e.g. (someEvent /at 2022-01-05 1900)
+ */
+
+public class AddCommand extends Command {
+
+    private final String details;
+
+    /**
+     * Constructs AddCommand object
+     *
+     * @param command command of the user input
+     * @param details details of the user input AFTER command is filtered
+     */
+    public AddCommand(String command, String details) {
+        super(command);
+        this.details = details;
+    }
+
+    /**
+     * Adds Tasks into the TaskRecords
+     *
+     * @param taskList stores the list of tasks
+     * @param ui       Object that responsible in returning necessary formatted String
+     *                 to print on the user interface
+     * @return String of suitable response according to the user input through BotUI object.
+     * @throws DukeException - thrown from Parser.extractDateTime methods.
+     * @see Parser - the details of the extratDateTime method throw DukeException

```

Comment:
Minor nitpick of misspelled method name.

Reply:
Noted! Thanks for the recommendation!